cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc

```diff
@@ -18,22 +18,24 @@ toc::[]
 //  - Connor Wooley
 //---
 
-link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities such as databases or message queues within the cluster.
 
-Users can install various ACK Operators directly from OperatorHub. This makes it relatively easy to get started and start using it with your applications. This controller is a component of the AWS Controller for Kubernetes project. This project is currently in developer preview.
+You can install various ACK Operators directly from OperatorHub. This makes it easy to get started and use the Operators with your applications. This controller is a component of the AWS Controller for Kubernetes project, which is currently in developer preview.
```

Comment:
Question (based on my limited knowledge of the relation between operators/controllers): Is this statement saying the specific controller is in developer review or the project itself (which this controller is a part of)? I guess the safe assumption would be the controller is also in developer preview, b/c it is part of the overall project? I guess my other question would be what is the difference between tech preview (which I believe is a term we often use) and "developer preview"? May not be related to this doc, just wondering if they are interchangeable.